pkg/cmd/cli/cmd/importimage.go

```diff
@@ -0,0 +1,139 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/fields"
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/watch"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/cli/describe"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	importImageLong = `Import tag and image information from an external Docker image repository.
+
+Only image streams that have a value set for spec.dockerImageRepository may
+have tag and image information imported.`
+
+	importImageExample = `  $ %[1]s import-image mystream`
+)
+
+// NewCmdImportImage implements the OpenShift cli import-image command.
+func NewCmdImportImage(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "import-image IMAGESTREAM",
+		Short:   "Imports tag and image information from an external Docker image repository.",
```

Comment:
Not sure what you mean. All this does is clear the annotation so the image import controller will process this stream again.